juniper_hyper/src/lib.rs

```diff
@@ -371,16 +371,6 @@ impl fmt::Display for GraphQLRequestError {
 }
 
 impl Error for GraphQLRequestError {
-    fn description(&self) -> &str {
-        match *self {
-            GraphQLRequestError::BodyHyper(ref err) => err.description(),
-            GraphQLRequestError::BodyUtf8(ref err) => err.description(),
-            GraphQLRequestError::BodyJSONError(ref err) => err.description(),
-            GraphQLRequestError::Variables(ref err) => err.description(),
-            GraphQLRequestError::Invalid(ref err) => err,
-        }
-    }
-
     fn cause(&self) -> Option<&dyn Error> {
```

Comment:
`cause` is also deprecated and replaced by `source`.

Reply:
Interesting... do you know why this warning isn't also being surfaced? And should I add that change to this PR or is it out of scope?

Reply:
Warning is triggered by usage (invocation) of deprecated method, not by its declaration.

Reply:
Ok, fixed `cause()` in f840576